.github/workflows/python-package.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 0 * * *'
+
+env:
+  MJKEY: ${{ secrets.MJKEY }}
+
+jobs:
+  test-docker:
+
```

Comment:
a name can also be added to the job so it will show up with that name in checks

Reply:
Yeah -- nice names are really helpful in the code review UI and cost nothing.